docker-registry/README.md

```diff
@@ -0,0 +1,27 @@
+Docker-registry
+===============
+
+```
+docker run -d \
+         --name ceph-docker-registry
+         -e AWS_BUCKET=mybucket \
+         -e AWS_KEY=myawskey \
+         -e AWS_SECRET=myawssecret \
+         -e AWS_HOST=myowns3.com \
+         -e AWS_SECURE=true \
+         -e AWS_PORT=80 \
+         -e AWS_CALLING_FORMAT=boto.s3.connection.OrdinaryCallingFormat \
```

Comment:
it is related to this:
http://ceph.com/docs/master/install/install-ceph-gateway/#add-wildcard-to-dns

when you don't have the wildcard.

common used when you want encryption but you don't have a wildcard cert